src/test/app/claims/models/claim.ts

```diff
@@ -1063,6 +1063,19 @@ describe('Claim', () => {
     })
   })
 
+  describe('handoffToCCBC', () => {
```

Comment:
It is possible to refactor this test case? We cannot be testing the value we assigned

Reply:
done